### DISCUSSION TITLE
@types/color no longer fitting.

### DISCUSSION THREAD

**Discussion by Vithanco** [DefinitelyTyped/DefinitelyTyped#63848] [Issues with a @types package]
some internals have changed. What was Color.array() is now Color.color

[a comment by an automated account is not shown]

**Comment by Vithanco** [comment]
ok, digging deeper, my statement might be wrong. I do have issues with Color that don't make sense.

**Comment by Vithanco** [comment]
I struggle for example to get a hex string from a Color. Typescript accepts the `aColor.hex()` but the runtime complains about  `aColor.hex is not a function`.
Similar for `array()` which looks good for Typescript, but at runtime I can only access `color` which is again not accepted by Typescript.

**Reply by BendingBender** [reply]
Can you please provide a complete and minimal example of what doesn't work but is allowed in TS?
